Library/Homebrew/cmd/cleanup.rb

```diff
@@ -33,6 +33,10 @@ def cleanup_args
   def cleanup
     args = cleanup_args.parse
 
+    if args.prune.present? && !Integer(args.prune, exception: false) && args.prune != "all"
+      odie "--prune= expects an integer or 'all'."
```

Comment:
```suggestion
      raise UsageError, "--prune= expects an integer or 'all'."
```

so `--help` will be printed